JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,15 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  const hours = Math.floor(seconds / 3600);
+  const minutes = Math.floor((seconds % 3600) / 60);
+  const remainingSeconds = seconds % 60;
+
+  const units = [hours, minutes, remainingSeconds];
+
+  const formattedUnits = units.map((unit) => unit.toString().padStart(2, '0'));
+
+  return formattedUnits.join(':');
```

Comment:
you can also return this directly right? but I understand why you separate this to have more readability 💯 

return timeUnits.map((unit) => unit.toString().padStart(2, '0')).join(':');